app/controllers/api/v1/orders_controller.rb

```diff
@@ -15,7 +15,12 @@ def create
     attributes = order_params
     @order = Order.create(attributes)
     order_items_params.each { |item| add_order_item(item[:menu_item], item[:quantity]) }
-    invalid_request unless @order.save
+    if @order.save
+      OrderNotifier.kitchen(@order).deliver_now
+      OrderNotifier.customer(@order).deliver_now
+    else
+      invalid_request unless @order.save
```

Comment:
Just one suggestion, instead of having the mail deliveries here, you could have them as an `after_create` callback in the model.